src/components/FilterDate/FilterDate.test.tsx

```diff
@@ -0,0 +1,83 @@
+import testArray from "../../utils/testArray";
+import * as React from "react";
+import * as Enzyme from "enzyme";
+import { mount } from 'enzyme';
+import * as Adapter from 'enzyme-adapter-react-16';
+import FilterDate from "./FilterDate";
+
+Enzyme.configure({
+	adapter: new Adapter()
+});
+
+jest.mock("@root/i18n", () => {
+	return {
```

Comment:
Codacy has a fix for the issue: [Replace `↹` with `··`](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9287389)
```suggestion
  return {
```